crates/rspack_binding_options/src/options/mod.rs

```diff
@@ -222,17 +222,20 @@ impl RawOptionsApply for RawOptions {
     );
     plugins.push(rspack_plugin_javascript::JsPlugin::new().boxed());
     plugins.push(rspack_plugin_javascript::InferAsyncModulesPlugin {}.boxed());
-    plugins.push(
-      rspack_plugin_devtool::DevtoolPlugin::new(rspack_plugin_devtool::DevtoolPluginOptions {
-        inline: devtool.inline(),
-        append: !devtool.hidden(),
-        namespace: output.unique_name.clone(),
-        columns: !devtool.cheap(),
-        no_sources: devtool.no_sources(),
-        public_path: None,
-      })
-      .boxed(),
-    );
+
+    if devtool.plugin_enabled() {
```

Comment:
`devtool.source_map()` looks better to me

Reply:
I didn't implement `'eval'` so don't need to care about `'eval'`, even if I need to implement it in the future I will add an another plugin to do that

Reply:
> `devtool.source_map()` looks better to me

source_map() has been exists and not works for devtool=eval